examples/non-kubernetes/aws-otel-ecs-ec2-deployment.yaml

```diff
@@ -48,15 +49,16 @@ Resources:
             - hostPort: 4317
               protocol: tcp
               containerPort: 4317
+            - hostPort: 9411
+              protocol: tcp
+              containerPort: 9411
             - hostPort: 55681
```

Comment:
Should we expose 4318 as well (the default one for http?)

Reply:
Exposed ;) Thanks!